.github/CHANGELOG.md

```diff
@@ -5,7 +5,20 @@
 * Added the ``SimplifiedTwoDesign`` template, which implements the circuit 
   design of `Cerezo et al. (2020) <https://arxiv.org/abs/2001.00550>`_. 
   [(#556)](https://github.com/XanaduAI/pennylane/pull/556)
-
+  
+  .. figure:: ../../_static/templates/layers/simplified_two_design.png
+    :align: center
+    :width: 40%
+    :target: javascript:void(0);
+
+* Added the ``BasicEntanglerLayers`` template, which is a simple layer architecture 
+  of rotations and CNOT nearest-neighbour entanglers. 
+  [(#555)](https://github.com/XanaduAI/pennylane/pull/555)
+  
+  .. figure:: ../../_static/templates/layers/basic_entangler.png
+    :align: center
+    :width: 40%
+    :target: javascript:void(0);
```

Comment:
This is a markdown file, not ReST! So the images need to use markdown syntax _or_ html. Probably html is best, i.e.,
```html
  <img src="https://pennylane.readthedocs.io/en/latest/_images/qaoa_layers.png"
  width=70%></img>
```

Reply:
Oh, of course. Strangely enough, it worked before...

Reply:
The simplified_two_design figure link will be broken until the PR is merged into master...